### PR TITLE
[plg_captcha_recaptcha]code style

### PR DESCRIPTION
**Pull request by wojsmol**
### Summary of Changes

Code style changes for plg_captcha_recaptcha
### Testing Instructions

code review

### CODE REVIEW

File: plugins/captcha/recaptcha/recaptcha.xml [hunk 1]
```diff
@@ -14,92 +14,123 @@
...
+					name="theme"
+					type="list"
+					label="PLG_RECAPTCHA_THEME_LABEL"
+					description="PLG_RECAPTCHA_THEME_DESC"
+					default="clean"
+					required="false"
+					showon="version:1.0"
+					filter=""
+					>
+					<option
```
Comment (andrepereiradasilva) [inline plugins/captcha/recaptcha/recaptcha.xml]: the changes in the options are not needed. Is not a very big line and makes it harder to read IMO.

File: plugins/captcha/recaptcha/recaptcha.xml [hunk 2]
```diff
@@ -14,92 +14,123 @@
...
+				<field
+					name="theme2"
+					type="list"
+					label="PLG_RECAPTCHA_THEME_LABEL"
+					description="PLG_RECAPTCHA_THEME_DESC"
+					default="light"
+					required="false"
+					showon="version:2.0"
+					filter="">
+					<option
```
Comment (andrepereiradasilva) [inline plugins/captcha/recaptcha/recaptcha.xml]: the changes in the options are not needed. Is not a very big line and makes it harder to read IMO.

File: plugins/captcha/recaptcha/recaptcha.xml [hunk 3]
```diff
@@ -14,92 +14,123 @@
...
+					name="size"
+					type="list"
+					label="PLG_RECAPTCHA_SIZE_LABEL"
+					description="PLG_RECAPTCHA_SIZE_DESC"
+					default="normal"
+					required="false"
+					showon="version:2.0"
+					filter="">
+					<option
+						value="normal"
```
Comment (andrepereiradasilva) [inline plugins/captcha/recaptcha/recaptcha.xml]: the changes in the options are not needed. Is not a very big line and makes it harder to read IMO.

**Comment by JoomliC** [comment]
I have tested this item :white_check_mark: successfully on 9fbdb1f17d186a2d0bc2788f4c0f77614979b751<br /><br />On code review<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10471">issues.joomla.org/joomla-cms/10471</a>.</sub>

**Comment by andrepereiradasilva** [comment]
I have tested this item :white_check_mark: successfully on 9fbdb1f17d186a2d0bc2788f4c0f77614979b751<br /><br />on code review<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10471">issues.joomla.org/joomla-cms/10471</a>.</sub>

**Comment by brianteeman** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10471">issues.joomla.org/joomla-cms/10471</a>.</sub>

**Pull request merged**